config/seata-config-core/src/main/java/io/seata/config/FileConfiguration.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.seata.config;
+
+import java.util.Set;
+
+import org.apache.seata.common.exception.NotSupportYetException;
+
+/**
+ * The type File configuration.
+ *
+ * Notes: used for Apache ShardingSphere and ConfigurationFactory
+ * 1.
+ * https://github.com/apache/shardingsphere/blob/master/kernel/transaction/type/base/seata-at/src/main/java/org
+ * /apache/shardingsphere/transaction/base/seata/at/SeataATShardingSphereTransactionManager.java
+ * 2.EnhancedServiceLoader.load(ExtConfigurationProvider.class).provide(configuration)
+ */
+public class FileConfiguration extends AbstractConfiguration {
```

Comment:
```suggestion
@Deprecated
public class FileConfiguration extends AbstractConfiguration {
```

Reply:
Based on the conclusions from the previous meeting, Deprecated annotations will be updated uniformly.